bucket/lxmusic.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "2.2.2",
+    "homepage": "http://lyswhut.github.io/lx-music-doc/",
+    "description": "a free & open source music finder",
+    "license": "Apache-2.0",
+    "shortcuts": [
+        [
+            "lx-music-desktop.exe",
+            "Lx Music Desktop"
+        ]
+    ],
+    "checkver": "github",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/lyswhut/lx-music-desktop/releases/download/v2.2.2/lx-music-desktop-v2.2.2-win_x86-green.7z",
+            "hash": "md5:199B741267CFF45AFC70DC2EDA114A92"
```

Comment:
To use md5 instead of sha256 here, you should specify where and how to capture in `autoupdate`, [reference](https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-customized-regex-in-autoupdatehash)

Reply:
![图片](https://github.com/chawyehsu/dorado/assets/49832311/3de70028-d52b-40f2-ba3f-e0db606a039a)

I have a problem with the File Md5 on the page. I don't know how to match the specified version for the architecture. Could you give me some advice? Can I configure the hash for each architecture? I didn't find any help in the Scoop wiki.

Reply:
The update checker can capture md5 value from specified page with RegEx
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#using-customized-regex-in-autoupdatehash
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#properties-of-autoupdatehash
https://github.com/ScoopInstaller/Scoop/wiki/App-Manifest-Autoupdate#hash-variables
In this case, use
```
"autoupdate": {
    "architecture": {
        "32bit": {
            "url": "https://github.com/lyswhut/lx-music-desktop/releases/download/$version/lx-music-desktop-$version-win_x86-green.7z",
            "hash": {
                "url": "https://github.com/lyswhut/lx-music-desktop/releases/tag/v$version",
                "regex": "$md5\\s+lx-music-desktop\\s+v$version\\s+win_x86\\s+green"
            }
        },
        "64bit": {
            "url": "https://github.com/lyswhut/lx-music-desktop/releases/download/$version/lx-music-desktop-$version-win_x64-green.7z",
            "hash": {
                "url": "https://github.com/lyswhut/lx-music-desktop/releases/tag/v$version",
                "regex": "$md5\\s+lx-music-desktop\\s+v$version\\s+win_x64\\s+green"
            }
        },
        "arm64": {
            "url": "https://github.com/lyswhut/lx-music-desktop/releases/download/$version/lx-music-desktop-$version-win_arm64-green.7z",
            "hash": {
                "url": "https://github.com/lyswhut/lx-music-desktop/releases/tag/v$version",
                "regex": "$md5\\s+lx-music-desktop\\s+v$version\\s+win_arm64\\s+green"
            }
        }
    }
}

```

Reply:
thanks for your help